maps/Nightmare/maps/LV624/scenario.json

```diff
@@ -1,19 +1,9 @@
 [
-{
-	"type": "pick", "name": "LV624 Potential Path",
-	"choices": [
-		{ "type": "def", "values": { "mainpath": "left"   } },
-		{ "type": "def", "values": { "mainpath": "bridge" } },
-		{ "type": "def", "values": { "mainpath": "right"  } }
-	]
-},
 {
 	"type": "pick", "name": "LV624 Global Event",
 	"choices": [
 		{ "weight": 2, "type": "def", "values": { "lvevent": "none"       } },
-		{ "weight": 4, "type": "def", "values": { "lvevent": "last_stand" } },
-		{ "weight": 2, "type": "def", "values": { "lvevent": "fallen_ship", "mainpath": "bridge"  } },
-		{ "weight": 2, "type": "def", "values": { "lvevent": "fallen_ship", "mainpath": "right" } }
```

Comment:
wait no don't remove these, roll only the lvevent, we still need fallen_ship to be an option

```
{ "weight": 4, "type": "def", "values": { "lvevent": "fallen_ship" } },
```

everything looks good otherwise 👍